.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: ruby
+script: docker-compose run --rm api bundle exec rake spec
+install:
+  - cp .env.example .env
+  - docker-compose run --rm --no-deps api bundle install
+  - docker-compose run --rm api bundle exec rake db:setup
```

Comment:
with the changes I made to the Dockerfile, the bundle install shouldn't be necessary